pl/lessons/basics/functions.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: page
+title: Funkcje
+category: basics
+order: 7
+lang: pl
+---
+
+W Elixirze tak, jak w wielu innych językach funkcyjnych, funkcje należą do bytów podstawowych (ang. _first class 
+citizen_). W tej lekcji poznamy rodzaje funkcji, różnice pomiędzy nimi oraz zastosowania.
```

Comment:
Is it a new line or diff is displayed in such way? This should be continuation of previous line.